### ISSUE TITLE
AndroidManifest.xml, additional properties for Application

### ISSUE THREAD

**Issue by DanBrooker** [HipByte/RubyMotion#181]
Hi,

I'd like to add additional properties into the AndroidManifest.xml.
There's no current way to include arbitrary properties into <application > or main <activity >
eg. android:screenOrientation="portrait", or android:logo="@drawable/logo"

android:logo for instance affects the visual launch state of the activity.

Maybe something like this?
```ruby
# Rakefile
app.manifest['application']['android:logo'] = '@drawable/logo'
```
and then
```ruby
# android.rb
android_manifest_txt << <<EOS
  <application android:label="#{App.config.name}"    ...
  #{ App.config.manifest['application'].map { |k,v| "#{k}=\"#{v}\""  }.join(" ") }
  >
EOS
```

Regards,
Dan

**Comment by DanBrooker** [comment]
I've just come across another reason to have a more flexible manifest 

example: push notifications
```xml
<application .... >

        ...
         
        <receiver
            android:name="com.google.android.gcm.GCMBroadcastReceiver"
            android:permission="com.google.android.c2dm.permission.SEND" >
            <intent-filter>
 
                <!-- Receives the actual messages. -->
                <action android:name="com.google.android.c2dm.intent.RECEIVE" />
                <!-- Receives the registration id. -->
                <action android:name="com.google.android.c2dm.intent.REGISTRATION" />
 
                <category android:name="com.androidexample.gcm" />
            </intent-filter>
        </receiver>
 
        <service android:name="com.androidexample.gcm.GCMIntentService" />
    </application>
```

**Comment by MarkVillacampa** [comment]
This is available since Rubymotion 3.6 with the addition of the app.manifest property.

http://www.rubymotion.com/developers/guides/manuals/android/project-management/#_providing_custom_values
